sklearn/linear_model/_least_angle.py

```diff
@@ -1291,6 +1308,21 @@ class LassoLars(Lars):
     [ 0.         -0.955...]
     """
 
+    _parameter_constraints = {
```

Comment:
`LassoLars` is inheriting from `Lars` so there is no need to repeat the shared parameter if they have the same values